ios/RN/RNCameraManager.m

```diff
@@ -198,43 +192,30 @@ + (NSDictionary *)faceDetectorConstants
 
 RCT_REMAP_METHOD(record,
                  withOptions:(NSDictionary *)options
+                 reactTag:(nonnull NSNumber *)reactTag
                  resolver:(RCTPromiseResolveBlock)resolve
                  rejecter:(RCTPromiseRejectBlock)reject)
 {
 #if TARGET_IPHONE_SIMULATOR
     reject(@"E_RECORDING_FAILED", @"Video recording is not supported on a simulator.", nil);
     return;
 #endif
-    [self.bridge.uiManager addUIBlock:^(__unused RCTUIManager *uiManager, NSDictionary<NSNumber *, UIView *> *viewRegistry) {
-        RNCamera *view = nil;
-        for (NSNumber *reactTag in viewRegistry) {
-            UIView *reactView = viewRegistry[reactTag];
-            if ([reactView isKindOfClass:[RNCamera class]]) {
-                view = (RNCamera *)reactView;
-                break;
-            }
-        }
-        if (!view) {
-            RCTLogError(@"Could not find RNCamera view on viewRegistry");
+    [self.bridge.uiManager addUIBlock:^(__unused RCTUIManager *uiManager, NSDictionary<NSNumber *, RNCamera *> *viewRegistry) {
+        RNCamera *view = viewRegistry[reactTag];
+        if (![view isKindOfClass:[RNCamera class]]) {
+            RCTLogError(@"Invalid view returned from registry, expecting RNCamera, got: %@", view);
         } else {
             [view record:options resolve:resolve reject:reject];
         }
     }];
 }
 
-RCT_EXPORT_METHOD(stopRecording)
+RCT_REMAP_METHOD(stopRecording, reactTag:(nonnull NSNumber *)reactTag)
```

Comment:
remap?

Reply:
Yes, when you receive a parameter you should use RCT_REMAP_METHOD.

Reply:
Great